test/tinycore.txt

```diff
@@ -2,8 +2,15 @@ These are binaries from
     http://tinycorelinux.net
 with the following patch applied:
 
+Source is available http://www.tinycorelinux.net/6.x/x86/release/src/
+
+curl -O http://www.tinycorelinux.net/6.x/x86/release/distribution_files/vmlinuz64
+# c4f46b5cd826d7a765a3e85439309cfc  vmlinuz64
+mv vmlinuz64 vmlinuz
+curl -O http://www.tinycorelinux.net/6.x/x86/release/distribution_files/core.gz
+mv core.gz initrd.gz
 mkdir initrd
-( cd initrd ; zcat ../initrd.gz | sudo cpio -idm )
-sudo sed -i '/^# ttyS0$/s#^..##' initrd/etc/securetty 
-sudo sed -i '/^tty1:/s#tty1#ttyS0#g' initrd/etc/inittab
+( cd initrd ; gzcat ../initrd.gz | sudo cpio -idm )
+sudo sed -i -e '/^# ttyS0$/s#^..##' initrd/etc/securetty 
+sudo sed -i -e '/^tty1:/s#tty1#ttyS0#g' initrd/etc/inittab
 ( cd initrd ; find | sudo cpio -o -H newc ) | gzip -c > initrd.gz
```

Comment:
Thanks. I'll send an updated PR later. Updating to Alpine isn't hard but I
can do that in a follow up.
On May 19, 2016 09:16, "Ian Campbell" <notifications@github.com> wrote:

> In test/tinycore.txt
> <https://github.com/docker/hyperkit/pull/23#discussion_r63875498>:
>
> >  mkdir initrd
> > -( cd initrd ; zcat ../initrd.gz | sudo cpio -idm )
> > -sudo sed -i '/^# ttyS0$/s#^..##' initrd/etc/securetty
> > -sudo sed -i '/^tty1:/s#tty1#ttyS0#g' initrd/etc/inittab
> > +( cd initrd ; gzcat ../initrd.gz | sudo cpio -idm )
> > +sudo sed -i -e '/^# ttyS0$/s#^..##' initrd/etc/securetty
> > +sudo sed -i -e '/^tty1:/s#tty1#ttyS0#g' initrd/etc/inittab
> >  ( cd initrd ; find | sudo cpio -o -H newc ) | gzip -c > initrd.gz
>
> Currently our release testing for hyperkit is what is in circle.yml (i.e.
> not very much). In the context of Docker for Mac it of course gets a whole
> bunch more but that is all specific to that environment and doesn't use the
> stuff here in test at all (that stuff is all inherited from xhyve).
>
> I'm very much in two minds about committing the binaries vs downloading at
> test time. The images are ~10M which isn't too much overall to either
> download or checkin, but in both cases over time it will add up to a lot of
> either network traffic or git repo size (which == network traffic for the
> clone).
>
> Downloading is probably my slight preference, it does add a dependency on
> an external resource, but it is also the easier decision to change our
> minds about since it's not as permanent as committing blobs to the repo.
>
> So, I think I've just about talked myself into saying we should download
> ephemerally ;-).
>
> BTW I'd like eventually to switch to using Alpine as our default Linux
> test case, since that more closely matches our downstream usage. Not
> something for you to worry about though.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/hyperkit/pull/23/files/2edf677bae7fb8062595b7b214cc57e040ca8903#r63875498>
>